assets/styles.css

```diff
@@ -0,0 +1,109 @@
+@import url("https://fonts.googleapis.com/css2?family=Zen+Old+Mincho&display=swap");
```

Comment:
- [ ] Please Make sure to follow the figure design as it is in the template right now your fonts, colors spacing and the general layout doesn't match the Figma design. The reason you should follow the design is that you may be asked to create some CSS that matches a design template exactly. please practice that ability by using this project Kindly 🙏.
The correct font, in this case, is `Poppins` I'm assuming you are using Template 1. so please match the design of the template.